pages/services/edge-lb/1.2/installing/index.md

```diff
@@ -17,35 +17,35 @@ Configure a service account and install the Edge-LB package using the instructio
 - You have access to [the remote Edge-LB repositories](https://support.mesosphere.com/hc/en-us/articles/213198586).
 
 **Limitations**
-- Currently, Edge-LB works only with DC/OS Enterprise in permissive mode on DC/OS 1.10, and permissive or strict mode on DC/OS 1.11 [security mode](/latest/security/ent/#security-modes). 
+- Edge-LB supports all [security modes] in DC/OS 1.11 and later. It supports Permissive, Disabled in DC/OS 1.10. DC/OS 1.9 or earlier is not supported.
 
 # Add Edge-LB package repositories
-The Edge-LB package comprises two components: 
+The Edge-LB package comprises two components:
 
-- The **Edge-LB API server** is a restful API that manages one or more Edge-LB pools. Each Edge-LB pool is a collection of load balancers. 
+- The **Edge-LB API server** is a restful API that manages one or more Edge-LB pools. Each Edge-LB pool is a collection of load balancers.
 
-- An **Edge-LB pool** can be used to launch one or more instances of a load balancer to create a single highly available load balancer. Currently the Edge-LB pool supports only HAProxy as a load balancer. 
+- An **Edge-LB pool** can be used to launch one or more instances of a load balancer to create a single highly available load balancer. Currently the Edge-LB pool supports only HAProxy as a load balancer.
 
-You must install universe repos for the Edge-LB API server and the Edge-LB pool in order to install Edge-LB. 
+You must install Universe repositories for the Edge-LB API server and the Edge-LB pool in order to install Edge-LB.
 
 <p class="message--note"><strong>NOTE: </strong>If your environment is behind a firewall or otherwise not able to access the public catalog, then you must use a local catalog.</p>
 
 
-## Obtain artifacts
+## Obtaining package artifacts
 
 
-If your cluster already has connectivity, you can obtain the artifacts directly. 
-
-If you do not have a cluster with connectivity, you will then need a customer service account with Mesosphere to download the artifacts for each of the repos from the <a href="https://support.mesosphere.com/hc/en-us/articles/213198586">Mesosphere customer support site</a>. 
+In order to install both packages, you need to obtain package artifacts. They can be downloaded from <a href="https://support.mesosphere.com/hc/en-us/articles/213198586">Mesosphere customer support site</a>. 
```

Comment:
I am just now learning that the edge-lb two components, that are available on two universes, are also distributed as exactly two packages. Maybe we should add.

`To install both packages (Edge-LB and Edge-LB Pools)`

I also propose removing `in order`